### PR TITLE
Special class tag for making text larger, specific to Katana

### PR DESCRIPTION
**Pull request by ScolderCreations**
I felt like this might be the start of making Katana-specific styling…
Not tested.

### CODE REVIEW

**Comment by Darth-Ness** [comment]
Nice! I like the idea of having tags supported in Katana that aren't supported in Blink and Gecko because then someone else other than Google gets to decide what the internet supports. By the way this tag will render slightly smaller than <h3>. Also, is there a way I can edit this draft

**Comment by Darth-Ness** [comment]
I also just updated your code. It didn't create a pull-request for some reason though.

**Comment by ScolderCreations** [comment]
> I also just updated your code. It didn't create a pull-request for some reason though.

it just changed it, you have permissions to edit my code without prs.

**Comment by Darth-Ness** [comment]
Okay, maybe make the tag around 30? Also is there a way to change to merge into dev and not master?

**Comment by ScolderCreations** [comment]
> Okay, maybe make the tag around 30? Also is there a way to change to merge into dev and not master?

I wanted it to merge into dev, so I’ll look and see…

File: src/renderMain.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ def startWindow(toRender, isOnline, fileName):
 
 def renderText(TRtext, ITR, TTR, C):
     from tkinter.font import Font
-    textSizes = ["<katanaclassbig>", 15, "<h1",23.9, "<h2", 17.9, "<h3", 15.9, "<h4", 14, "<h5", 9.9, "<h6", 8, "<p", 12]
+    textSizes = ["<katanaclassbig>", 15, "<katanaclassbig2>",20,"<katanaclassbig3>",30, "<h1",23.9, "<h2", 17.9, "<h3", 15.9, "<h4", 14, "<h5", 9.9, "<h6", 8, "<p", 12]
```
Comment (ScolderCreations) [inline src/renderMain.py:34]: this is to allow more customization for sizes

Comment (Darth-Ness) [reply src/renderMain.py:34]: We should see if we can come up with some more original names. Maybe "big", "large", and "extralarge"? But the idea of having more font sizes without css is a good one.

**Comment by ScolderCreations** [comment]
I’m thinking of closing this if someone makes code to support CSS: Support CSS is way better than custom tags. Someone claimed to have done so, so this PR may have to be rejected.

**Comment by Darth-Ness** [comment]
> I’m thinking of closing this if someone makes code to support CSS: Support CSS is way better than custom tags. Someone claimed to have done so, so this PR may have to be rejected.

I'm not planning to support font-size soon. As it doesn't really seem to be as necessary as other things in CSS.

**Pull request merged**